.github/workflows/auto-test.yml

```diff
@@ -0,0 +1,51 @@
+name: Automatic +ve/-ve Testing
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Set up Python 3.9
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.9
+
+    - name: Install the package
+      run: |
+        python setup.py install
+
+    - name: svmap tests
+      run: |
+        pushd autotest
+        ./svmap.sh
+        popd
+
+    - name: +ve tests for svwar
```

Comment:
tests for svwar and svcrack need to be put in a separate file so that they can be run locally too for testing purposes